modules/utility/preview/glow/config.nix

```
@@ -0,0 +1,28 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}: let
+  inherit (lib) nvim mkIf mkMerge mkBinding;
+
+  cfg = config.vim.utility.preview.glow;
+  self = import ./glow.nix {
+    inherit lib config pkgs;
+  };
+  mappings = self.options.vim.utility.preview.glow.mappings;
+in {
+  config = mkIf cfg.enable {
+    vim.startPlugins = ["glow-nvim"];
+
+    vim.maps.normal = mkMerge [
+      (mkBinding cfg.glow.mappings.openPreview ":Glow<CR>" mappings.openPreview.description)
```

```suggestion
      (mkBinding cfg.mappings.openPreview ":Glow<CR>" mappings.openPreview.description)
```
Already inside the glow attribute.